### DISCUSSION TITLE
Academic benefits not available after 5 days

### DISCUSSION THREAD

**Discussion by MaxOrtGit** [community/community#138290] [GitHub Education]
### Hi everyone,

My application was approved on September 5th but I am still getting the message "Your request was approved. Your benefits will be available soon."

The link said after 5 days to send a message here to hopefully get it sorted.

**Comment by davevad93** [comment]
Hi @MaxOrtGit ,

Don't worry you'll have your benefits soon, it usually takes a few days but the waiting time may vary depending on different factors, such as the volume of requests, in which case it may take longer. 

 When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @MaxOrtGit was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by codeguyakash** [comment]
wait for some time and also try : https://education.github.com/pack
